packages/flutter/lib/src/widgets/single_child_scroll_view.dart

```diff
@@ -276,7 +276,7 @@ class SingleChildScrollView extends StatelessWidget {
         child: scrollable,
         onNotification: (ScrollUpdateNotification notification) {
           final FocusScopeNode currentScope = FocusScope.of(context);
-          if (notification.dragDetails != null && !currentScope.hasPrimaryFocus && currentScope.hasFocus) {
+          if (!currentScope.hasPrimaryFocus && currentScope.hasFocus) {
```

Comment:
My understanding of what's happening here is that the widget is now always unfocused when it is being scrolled? If that's correct, might there be cases where that behavior is undesired?

Reply:
@victorsanni Not that I am aware of, now it is as good as scroll with a drag which removes focus from the field when scrollbar is dragged. I found another issue #150048 which is similar to this and there is a discussion on this.